shared/src/commonMain/kotlin/io/customer/shared/sdk/meta/Region.kt

```diff
@@ -0,0 +1,31 @@
+package io.customer.shared.sdk.meta
+
+/**
+ * Region that your Customer.io Workspace is located in.
+ *
+ * The SDK will route traffic to the correct data center location depending on the [Region] that
+ * you use.
+ */
+sealed class Region(val code: String, internal val trackingURL: String) {
```

Comment:
See my other comments on lack of iOS support for `sealed class`.

Reply:
See Shahroz's response for sample of generated classes